extensions/cactus-plugin-htlc-eth-besu/src/main/contracts/contracts/HashTimeLock.sol

```diff
@@ -0,0 +1,165 @@
+pragma solidity ^0.7.3;
+pragma experimental ABIEncoderV2;
+
+contract HashTimeLock {
```

Comment:
There are not function to get  the balance of the HTLC deposit. I believe it is necessary to be checked by counter party.